src/compounds/RequestList/RequestList.jsx

```diff
@@ -9,7 +9,7 @@ const RequestList = ({ requests }) => (
     <Title title='My Requests' size='medium' />
     <div className='rounded overflow-hidden'>
       {requests.length === 0 ? (
-        <p className='mt-2'>
+        <p className='mt-2 no-requests'>
```

Comment:
can you add a comment here stating that this class is purely for testing purposes in the webstore? otherwise it may accidentally get removed in the future if someone looks at it and realizes it isn't doing anything.

Reply:
done!